server/session.go

```
@@ -0,0 +1,280 @@
+		newSessionTools[k] = v
+	}
+
+	// Remove specified tools
+	for _, name := range names {
+		delete(newSessionTools, name)
+	}
+
+	session.SetSessionTools(newSessionTools)
+
+	// Send notification only to this session
+	s.SendNotificationToSpecificClient(sessionID, "notifications/tools/list_changed", nil)
```

golangci-lint (1.64.8)

Error return value of `s.SendNotificationToSpecificClient` is not checked

(errcheck)